infra/promtail/promtail-config.yaml

```diff
@@ -0,0 +1,26 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: https://802939:glc_eyJvIjoiMTA0NTcyMCIsIm4iOiJzdGFjay04NDg0ODQtaGwtd3JpdGUtbG9jYWx0ZXN0aW5nIiwiayI6IjB1MWNHOTMxVHRXQzJpTVR5cTczMUs3ViIsIm0iOnsiciI6InByb2QtZXUtd2VzdC0yIn19@logs-prod-012.grafana.net/loki/api/v1/push
```

Comment:
Is that some kind of an API key? Isn't it a bit problematic to be checked into the source code? It means everyone will log to our cloud instance, right?

Reply:
Yes, this is an access token to push logs to Loki where we have 50 GB of log storage with 2 week retention. If it exceeds that then I guess it will lower the retention.

I agree on checking the full key in the code, I originally did not but then I saw that unfortunately that `promtail` has marked `-client.url` as deprecated :(. Maybe we should just ignore that (or alternatively use a template yaml).

Reply:
Moved this to a repository secret